packages/react-dom/src/__tests__/ReactDOMServerSuspense-test.internal.js

```diff
@@ -41,7 +41,7 @@ function initModules() {
   };
 }
 
-const {itThrowsWhenRendering, resetModules, serverRender} =
+const {resetModules, itThrowsWhenRendering} =
```

Comment:
@gnoff line 38 is still using `ReactDOM` instead of `ReactDOMClient` so the `itThrowsWhenRendering` tests are still using the legacy renderer.

If you pass `ReactDOMClient` here:
```js
return {
    ReactDOMClient,
    ReactDOMServer,
    ReactTestUtils,
  };
```

Then we switch to `hyrdateRoot` in the helpers: 

https://github.com/facebook/react/blob/6054be9c86f2e01e5e24c1fe2182479fec8667e4/packages/react-dom/src/__tests__/utils/ReactDOMServerIntegrationTestUtils.js#L55-L63

Reply:
Ah I didn't realize the I had to return the client version for the utils to use it. I took another look and the thing the `itThrowsWhenRendering` was asserting actually is wrong now for concurrent roots on the client so really the test only should be asserting on server behavior (also probably what the test was written for originally given the filename). I just ripped out the test utils entirely now and it only uses the underlying modules directly.

Thanks for flagging!